pycbc/events/stat.py

```diff
@@ -1174,26 +1182,54 @@ def coinc_multiifo(self, s, slide, step, to_shift,
         benchmark_logvol = s[0][1]['benchmark_logvol']
         network_logvol -= benchmark_logvol
 
+        # Use prior histogram to get Bayes factor for signal vs noise given
+        # time, phase and SNR differences between IFOs
+        # First get signal term logr_s
         stat = {ifo: st for ifo, st in s}
         logr_s = self.logsignalrate_multiifo(stat,
                                              slide * step, to_shift)
 
-        loglr = logr_s + network_logvol - ln_noise_rate
+        # Calculate noise term of phase-time-amplitude Bayes factor, logr_n
+        noise_twindow = coinc_rate.multiifo_noise_coincident_area(
+                            self.hist_ifos, kwargs['time_addition'])
+        # Calculate the volume of the histogram which could be filled by
+        # noise coincidences. Not all bins may be occupied due to
```

Comment:
i'd suggest not talking about 'the histogram' here - a histogram is just a clunky way of calculating/storing a density over some space. what is being calculated is the volume of phase/time/amplitude space. 

I'll edit via the web interface since other points look fine